emails/views.py

```diff
@@ -501,6 +503,9 @@ def _sns_message(message_json):
     address.save(
         update_fields=['num_forwarded', 'last_used_at']
     )
+    # successful email relayed count number of trackers
+    if sample_is_active('tracker-sample') and html_content:
```

Comment:
👍 waffle samples are a good choice for this

Reply:
Thanks!